### PR TITLE
Add linode cloud example

### PR DESCRIPTION
**Pull request by sanyer**

### CODE REVIEW

File: bin/shared_linode.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+if [ -z "$LINODE_TOKEN" ]
```
Comment (ianmiell) [inline bin/shared_linode.sh:6]: I will fetch the value from the vault, but can you add the linode test to the GitHub workflow file? Let me know if you need help with this.

**Review by ianmiell — COMMENTED** [review]
Just one comment here.

**Comment by ianmiell** [comment]
The tests are failing with the INDEX.md file - again, let me know if you need help with this.

File: bin/shared_linode.sh (hunk 1 above)
Comment (ianmiell) [reply bin/shared_linode.sh:6]: The secret has been aded as LINODE_TOKEN

**Comment by ianmiell** [comment]
@sanyer I rebased this to get ahead of the testing, sorry if this gets in your way.

**Comment by sanyer** [comment]
I'll use your example of tests for the next clouds I'm gonna bring in.
Sorry for lack of response for last days.

**Pull request merged**